docs/website/docs/dlt-ecosystem/verified-sources/pg_replication.md

```diff
@@ -0,0 +1,253 @@
+
+2. `WITH LOGIN REPLICATION;` does not work on RDS, instead do:
+    
+    ```sql
+    GRANT rds_replication TO replication_user;
+    ```
+    
+3. Do not fallback to non-SSL connection by setting connection parameters:
+    
+   ```toml
+   sources.pg_replication.credentials="postgresql://loader:password@host.rds.amazonaws.com:5432/dlt_data?sslmode=require&connect_timeout=300"
+   ```
+### Initialize the verified source
+
+To get started with your data pipeline, follow these steps:
+
+1. Enter the following command:
+    
+   ```sh
+   dlt init pg_replication duckdb
+   ```
+    
+   It will initialize [the pipeline example](https://github.com/dlt-hub/verified-sources/blob/master/sources/pg_replication_pipeline.py) with a Postgres replication as the [source](https://dlthub.com/docs/general-usage/source) and [DuckDB](https://dlthub.com/docs/dlt-ecosystem/destinations/duckdb) as the [destination](https://dlthub.com/docs/dlt-ecosystem/destinations).
+    
+    
+2. If you'd like to use a different destination, simply replace `duckdb` with the name of your preferred [destination](https://dlthub.com/docs/dlt-ecosystem/destinations).
+    
+3. This source uses `sql_database` source, you can init it as follows:
+    
+   ```sh
+   dlt init sql_database duckdb
+   ```
+    
+4. After running these two commands, a new directory will be created with the necessary files and configuration settings to get started.
+   
+   
+   For more information, read the guide on [how to add a verified source](https://dlthub.com/docs/walkthroughs/add-a-verified-source).
+
+   :::note
+   You can omit the `[sql.sources.credentials]` section in `secrets.toml` as it is not required.
+   :::
+
+### Add credentials
+
+1. In the `.dlt` folder, there's a file called `secrets.toml`. It's where you store sensitive information securely, like access tokens. Keep this file safe.
+    
+   Here's what the `secrets.toml` looks like:
+    
+   ```toml
+   [sources.pg_replication.credentials]
+   drivername = "drivername" # please set me up!
+   database = "database" # please set me up!
+   password = "password" # please set me up!
+   username = "username" # please set me up!
+   host = "host" # please set me up!
+   port = 0 # please set me up! 
+   ```
+    
+2. Credentials can be set as shown above. Alternatively, you can provide credentials in the `secrets.toml` file as follows:
+    
+   ```toml
+   sources.pg_replication.credentials="drivername://username@password.host:port/database"
+   ```
+
+3. Finally, follow the instructions in [Destinations](https://dlthub.com/docs/dlt-ecosystem/destinations/) to add credentials for your chosen destination. This will ensure that your data is properly routed.
+
+For more information, read the [General Usage: Credentials.](https://dlthub.com/docs/general-usage/credentials)
```

Comment:
we don;t have General Usage section anymore, replace it only with `Configuration`

Reply:
updated